### ISSUE TITLE
Replace each() by foreach()

### ISSUE THREAD

**Issue by shellamaebailio** [Grandt/PHPePub#56]
This issue still exists on the latest version 4.0.7.1

**Comment by vipulchauhanmt04** [comment]
Yes right, I have faced this issue right now. This issue still exists on the latest version 4.0.7.1. 
Can you fix this issue on the package?
If the issue was not fixed then Please solve this issue on the package and update it here.
